azuredeploy.json

```diff
@@ -53,21 +64,33 @@
         "skuCode": "[parameters('skuName')]",
         "numberOfWorkers": "[parameters('numberOfInstances')]",
         "linuxFxVersion": "NODE|lts",
-        "hostingPlanName": "[concat('hpn-', parameters('siteName'))]",
+        "hostingPlanNameLinux": "[concat('hpn-', parameters('siteName'))]",
+        "hostingPlanNameWin": "[concat('hpn-win-', parameters('siteName'))]",
         "repoURL": "https://github.com/microsoft/HealthBotContainerSample.git",
-        "branch": "master"
+        "branch": "master",
+        "kind": "[if(equals(parameters('serverKind'), 'windows'), 'app', 'linux')]",
+        "linuxSiteName": "[concat(parameters('siteName'), 'linux')]",
+        "windowsSiteName": "[concat(parameters('siteName'), 'windows')]",
+        "WinSkuCode": "[parameters('skuName')]",
+        "WinSku": "Standard",
+        "workerSize": "0",
+        "workerSizeId": "0",
+        "hostingEnvironment": "",
+        "nodeVersion": "12.13.0",
```

Comment:
Can we use LTS instead?

Reply:
tbh, I never found a complete list of possible values and I took this one from a template I generated from the portal.